### ISSUE TITLE
Pipeline getting hung up near final steps

### ISSUE THREAD

**Issue by bblarsen-sci** [dms-vep/dms-vep-pipeline-3#77]
I have been having an issue where the pipeline will not go to completion. Inspecting the log files it looks like it finishes with `summary.txt`, but then `cp_html_summary_faceted.txt` and `cp_html_summary_overlaid.txt` remain empty and the pipeline will keep running until I either manually terminate it or the job is killed after running for many hours. I am not sure if this is a unique problem for me (if I have some problems in my config or input files that is causing this) or it's an issue for others. My repo in `/computational_notebooks/blarsen/2023/Nipah_Malaysia_RBP_DMS` is repeatedly having this issue. 

It's not a huge issue since it still makes all the intermediate files I am most interested in, but it will not generate the final docs file `index.html`. I think this problem arose when I updated to `dms-vep-pipeline-3 v3.4.7`.

**Comment by jbloom** [comment]
@bblarsen-sci Definitely I can look at this, but can you push the latest version of `Nipah_Malaysia_RBP_DMS` to GitHub so I can test with that? The current version on GitHub has `dms-vep-pipline-3` version 3.4.6, but above you indicate issue is with 3.4.7.

**Comment by bblarsen-sci** [comment]
Yep, my apologies, just pushed. Also, I'm not positive its an issue with v3.4.7, I think the issue arose when I updated the submodule but I'm not 100% certain.

**Comment by jbloom** [comment]
@bblarsen-sci, I'm not sure of the problem, but it worked for me. What if you just `rm -rf docs` to remove the `./docs` folder from your repo and try again?

**Comment by bblarsen-sci** [comment]
@jbloom thanks for checking! I tried that, still getting hung. I'm gonna try `rm -rf results`

**Comment by bblarsen-sci** [comment]
Its working now. Not sure what issue was. Closing.
